### DISCUSSION TITLE
data package identifier spec - GitHub example incorrect?

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [frictionlessdata/frictionlessdata.io#643] [Q&A] [answered]
Given 

> ## Identifier Object Structure
> The object structure looks like:
>
> ```
> {
>  // URL to base of the Data Package
>  // This URL should *always* have a trailing slash ('/')
>  url: ...
>

Shouldn't the GitHub example

> A GitHub URL:
>
>       http://github.com/datasets/gold-prices
>
>   resolves to:
>
>       https://raw.githubusercontent.com/datasets/gold-prices/master/datapackage.json

Use `http://github.com/datasets/gold-prices/` with a trailing slash because, "URL should *always* have a trailing slash ('/')" ?

**Comment by rufuspollock** [comment, marked as answer]
Good spot though I think the example is in fact correct: the parsed spec should have the url with trailing slash but the input identifier string need not have a trailing slash. Hope that makes sense.
